packages/techdocs-common/src/stages/generate/techdocs.ts

```diff
@@ -136,7 +153,7 @@ export class TechdocsGenerator implements GeneratorBase {
       this.logger.debug(
         `Failed to generate docs from ${inputDir} into ${outputDir}`,
       );
-      this.logger.error(`Build failed with error: ${log}`);
+      childLogger.error(`Build failed with error: ${log}`);
```

Comment:
The frontend build logs do not contain the actual error that is being thrown below. The preferred option would be to have any errors thrown in this scope also being added to the child logger. (If that's not possible, we should at least log the error in all cases before `throw`ing and exiting the scope)

Reply:
I also noticed that there are more logs that are not yet captured. For example these `Step 1 of 3: Preparing docs for entity ...` logs or the exception that e.g. docker is not available. The question is whether we want to pipe all these to the event stream and what kind of log messages we want to have in the backend logs. One option could be to skip logging anything to the default logger until we get an exception. Only then, print all previously captured logs. The scaffolder has these persisted in the database as an alternative, but I'm not sure if we want to go that far for TechDocs.